src/App.tsx

```diff
@@ -9,7 +9,10 @@ const App = () => {
 	const [error, setError] = useState(false);
 
 	const baseUrl =
-		process.env.REACT_APP_DESCOPE_BASE_URL || window.location.origin;
+		process.env.REACT_APP_DESCOPE_BASE_URL ??
```

Comment:
combining nullish operator and ternery expression in oneliner is a bit confusing imo

1. use `process.env.REACT_APP_DESCOPE_BASE_URL`
2. if nullish, than if the other conf exists - use origin, otherwise undefine

I would consider simplify this, we can use only `REACT_APP_DESCOPE_BASE_URL` and set it up in the proper places